rocAL/rocAL_pybind/rocal_pybind.cpp

```diff
@@ -153,6 +166,30 @@ namespace rocal
             output_tensor.copy_data(ptr);
             }
             )
+
+            .def("copy_data_numpy", [](rocalTensor &output_tensor, py::array_t<unsigned char> array) {
+                auto buf = array.request();
+                unsigned char *ptr = (unsigned char *)buf.ptr;
+                output_tensor.copy_data((void *)ptr);
+
+            },
+            "idx"_a,
+            R"code(
+            Returns a rocal tensor at given position `i` in the rocalTensorlist.
+            )code",
+            py::keep_alive<0, 1>())
```

Comment:
Why have we not added py value policy as reference here?
Please change to py::return_value_policy::reference instead of py::keep_alive<0, 1>()